pyaerocom/io/test/test_read_mscw_ctm.py

```diff
@@ -37,6 +37,13 @@ def test_read_emep_read_var(path_emep):
         r = ReadMscwCtm(data_dir=data_dir)
         r.read_var('vmro3', 'daily')
 
+@testdata_unavail
+def test_ts_type_dict(path_emep):
+    data_dir = path_emep['data_dir']
+    r = ReadMscwCtm(data_dir=data_dir)
+    data = r.read_var('vmro3', ts_type={'vmro3':'daily'})
+    assert data.ts_type == 'daily'
```

Comment:
I still do not understand why one would pass ts_type as dict in `read_var` and if this happens somewhere in the higher level code, it should probably be fixed there (if possible). @ejgal can you check that?

Reply:
It's from Colocator:
```
model_ts_type_read : :obj:`str` or :obj:`dict`, optional
        may be specified to explicitly define the reading frequency of the
        model data. Not to be confused with :attr:`ts_type`, which specifies
        the frequency used for colocation. Can be specified variable specific
        by providing a dictionary.
```
This can be passed all the way to the reader in some cases.
In ReadGridded._eval_vert_which there is also a check for ts_type being a dict, so there might be a usecase for this.

We could make sure to extract the ts_type for the variable with:
```
if isinstance(kwargs['ts_type'], dict):
    kwargs['ts_type'] = kwargs['ts_type'][var_name]
```
right before calling read_var.

Doing this does not break any of the tests at least.

Reply:
Should I try to fix this as proposed, or do we leave it for now @jgliss?

Reply:
Feel free to update as proposed, but please make an issue for it, so I can check at some point later what was the idea behind it :) ? It looks like this is for some edge cases and I am not sure anymore. But it seems it is also placed wrongly in if it is evaluated in `ReadGridded._eval_vert_which` ...

Reply:
Done: #276